src/luascript.cpp

```diff
@@ -11550,6 +11551,18 @@ int32_t LuaScriptInterface::luaItemTypeGetDecayId(lua_State* L)
 	return 1;
 }
 
+int32_t LuaScriptInterface::luaItemTypeGetRequiredLevel(lua_State* L)
+{
+	// itemType:getRequireLevel()
+	const ItemType* itemType = getUserdata<const ItemType>(L, 1);
+	if (itemType) {
+		lua_pushnumber(L, itemType->minReqLevel);
+	}else {
```

Comment:
Code formatting is important :). BTW. Easiest way to fix it is to push a new commit and then do rebase -i HEAD~2. Pick the first commit and squash the second, then change the commit message et voila!